task/prefetch-dependencies-oci-ta/0.1/README.md

```diff
@@ -18,6 +18,21 @@ https://github.com/containerbuildsystem/cachi2#basic-usage.
 |ociArtifactExpiresAfter|Expiration date for the trusted artifacts created in the OCI repository. An empty string means the artifacts do not expire.|""|false|
 |ociStorage|The OCI repository where the Trusted Artifacts are stored.||true|
 
+### Config file usage
+
+Config file must be passed as a YAML string. For all available config options please check [available configuration parameters]("https://github.com/containerbuildsystem/cachi2?tab=readme-ov-file#available-configuration-parameters") page.
+
+Example of setting timeouts:
+
+```yaml
+params:
+    - name: config-file-content
+      value: |
+         ---
+         requests_timeout: 300
```

Comment:
Just an FYI (not a request to change anything either in cachi2 or the docs): the timeout has a very different meaning for `requests` and for `aiohttp`.

For `requests`, it's the timeout for establishing the connection and the timeout between receiving some bytes from the server https://requests.readthedocs.io/en/latest/user/advanced/#timeouts

For `aiohttp`, it's the overall timeout for the whole download

As I found out in this flame-war of a thread https://github.com/aio-libs/aiohttp/issues/2249

Reply:
Yes it is, but I really didn't want to bother with connection and read timeout values in the aiohttp structure, because realistically the user cares only about the operation as a unit and from that POV the timeout here behaves the same in both cases, it's a cumulative value.

Reply:
> (is it that very different though?)

My understanding was that `requests` with a timeout of 1 minute will happily keep downloading for an hour if the server sends something every minute

While `aiohttp` will abort after a minute

Reply:
This is wrong PR to discuss this :D